src/layout/RepeatingGroup/Pagination/RepeatingGroupPagination.tsx

```diff
@@ -35,10 +35,10 @@ export function RepeatingGroupPagination(props: RepeatingGroupPaginationProps) {
     return null;
   }
 
-  return <_RepeatingGroupPagination {...props} />;
+  return <RGPagination {...props} />;
 }
 
-function _RepeatingGroupPagination({ inTable = true }: RepeatingGroupPaginationProps) {
+function RGPagination({ inTable = true }: RepeatingGroupPaginationProps) {
```

Comment:
Personally not a fan of abbreviations in naming, suggest going with RepeatingGroupPagination :)

Reply:
E9!